modules/images-allow-pods-to-reference-images-from-secure-registries.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+// * openshift_images/using-image-pull-secrets
+
+[id="images-allow-pods-to-reference-images-from-secure-registries_{context}"]
+= Allowing Pods to reference images from other secured registries
+
+The  `.dockercfg` file `$HOME/.docker/config.json` for Docker
```

Comment:
It's easy to cause misunderstanding here, ”.dockercfg“  “$HOME/.docker/config.json”